grakn-engine/src/main/java/ai/grakn/engine/backgroundtasks/distributed/ClusterManager.java

```diff
@@ -70,6 +72,10 @@ public ClusterManager() {
             LOG.debug("Starting Cluster manager on " + ENGINE_ID);
 
             startZookeeperConnection();
+
+            //TODO start state storage, decide using reflection
```

Comment:
We are going to be maintaining both. The InMemory will be used for testing non-engine tests that require engine to be running (GraqlShell, Migration and Analytics), which should speed tests up significantly. Boris also suggested that our single-machine dist could use it.